client/README.md

```diff
@@ -36,10 +38,41 @@ npm run build
 npm run start
 ```
 
-Open your browser to `localhost:8000`.
+This will show you the default version of the app, where you can "buy" certain
+available products.
+
+Open your browser to `localhost:8081` to explore more.
+
+### Client variants
+
+To display the "Avocart" version of the client.
```

Comment:
The "Avocart" version of Avocano extends the web app to show implementation of a shopping cart and checkout process, key requirements for ecommerce sites. 

To display the "Avocart" version of the client: